pkg/metrics/pool_test.go

```diff
@@ -114,3 +115,29 @@ pool_put_operation_total{name="foo",to="destroyed"} 1
 		})
 	}
 }
+
+func TestPoolCollectors(t *testing.T) {
```

Comment:
Hi @yashpandey06 this test doesn't seem to have any real test meaning, so I don't think it needs to be added.